### PR TITLE
Enable CD

### PR DESCRIPTION
**Pull request by batmat**
See https://www.jenkins.io/doc/developer/publishing/releasing-cd/

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,22 +8,26 @@
...
   </scm>
-  
+
   <properties>
+    <changelist>999999-SNAPSHOT</changelist>
+    <gitHubRepo>jenkinsci/javadoc-plugin</gitHubRepo>
     <bom>2.249.x</bom>
     <jenkins.version>2.249.3</jenkins.version>
     <java.level>8</java.level>
+    <gitHubRepo>jenkinsci/javadoc-plugin</gitHubRepo>
```
Comment (timja) [inline pom.xml:30]: github repo is in properties twice
```suggestion
```

**Review by timja — APPROVED** [review]
lgtm apart from duplicated property

File: pom.xml (hunk 1 above)
Comment (batmat) [reply pom.xml:30]: You're right :facepalm:

**Review by timja — APPROVED** [review]

**Comment by jglick** [comment]
Again remember that the [latest release](https://github.com/jenkinsci/javadoc-plugin/releases/tag/217.v905b_86277a_2a_) contains uncategorized changes, though since there was already https://github.com/jenkinsci/javadoc-plugin/releases/tag/javadoc-1.6 it is likely that all of these at least were _supposed_ to be here. This was missing a label, and I guess #42 was using an unrecognized label `maintenance` which ought to be deleted.

**Comment by olamy** [comment]
PR here https://github.com/jenkinsci/.github/pull/67

sounds logical to have entries with label `maintenance` going to section with title `Maintenance`

**Pull request merged**